modules/nf-core/ashlar/Dockerfile

```diff
@@ -0,0 +1,27 @@
+FROM ubuntu:22.04
```

Comment:
I don't think we need the Dockerfile in here?

Reply:
In the [Software Requirements section](https://nf-co.re/developers/modules#software-requirements) of the [DSL2 Modules documentation](https://nf-co.re/developers/modules), point 5 says:
> If the software is not available on Bioconda a Dockerfile MUST be provided within the module directory. We will use GitHub Actions to auto-build the containers on the GitHub Packages registry.

I have a question up on their slack about what the `container` call should be, but they haven't responded yet.